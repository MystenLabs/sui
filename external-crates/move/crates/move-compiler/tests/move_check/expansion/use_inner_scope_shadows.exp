error[E01002]: unexpected token
   ┌─ tests/move_check/expansion/use_inner_scope_shadows.move:12:5
   │
12 │     spec schema Foo<T> {
   │     ^^^^
   │     │
   │     Unexpected 'spec'
   │     Expected a module member: 'use', 'friend', 'const', 'fun', or 'struct'

