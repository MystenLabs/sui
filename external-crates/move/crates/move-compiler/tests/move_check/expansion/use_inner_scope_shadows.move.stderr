[error] external-crates/move/crates/move-compiler/tests/move_check/expansion/use_inner_scope_shadows.move: Error: tree-sitter failure on 
[error] ```
[error] address 0x2 {
[error] module M {
[error]     struct S1 has drop { b: bool }
[error]     struct S2 has drop { u: u64 }
[error]     struct S3 has drop { a: address }
[error]     fun check(): bool {
[error]         false
[error]     }
[error]     fun num(u: u64): u64 {
[error]         u + 1
[error]     }
[error]     spec schema Foo<T> {
[error]         ensures true;
[error]     }
[error] 
[error]     fun t<T>(): S3 {
[error]         use 0x2::M::{check as num, num as t, S1 as S2, S2 as Foo, S3 as T};
[error]         num() && true;
[error]         t(0) + 1;
[error]         S2 { b: false };
[error]         Foo { u: 0 };
[error]         T { a: @0x0 }
[error]     }
[error] 
[error]     fun t2<T>(x: T) {
[error]         {
[error]             use 0x2::M::{check as num, num as t, S1 as S2, S2 as Foo, S3 as T};
[error]             num() && true;
[error]             t(0) + 1;
[error]             S2 { b: false };
[error]             Foo { u: 0 };
[error]             T { a: @0x0 }
[error]         };
[error]         check() && true;
[error]         num(0) + 1;
[error]         t2<T>(x);
[error]         S1 { b: false };
[error]         S2 { u: 0 };
[error]         S3 { a: @0x0 };
[error]     }
[error] }
[error] }
[error] 
[error] ```
[error]     at checkErrorsCb (/Users/tedks/Projects/mysten/sui/external-crates/move/tooling/prettier-move/out/printer.js:30:23)
[error]     at Object.print (/Users/tedks/Projects/mysten/sui/external-crates/move/tooling/prettier-move/out/printer.js:39:16)
[error]     at callPluginPrintFunction (file:///Users/tedks/Projects/mysten/sui/node_modules/prettier/index.mjs:20422:20)
[error]     at printAstToDoc (file:///Users/tedks/Projects/mysten/sui/node_modules/prettier/index.mjs:20377:22)
[error]     at async coreFormat (file:///Users/tedks/Projects/mysten/sui/node_modules/prettier/index.mjs:20687:14)
[error]     at async formatWithCursor (file:///Users/tedks/Projects/mysten/sui/node_modules/prettier/index.mjs:20885:14)
[error]     at async formatFiles (file:///Users/tedks/Projects/mysten/sui/node_modules/prettier/internal/cli.mjs:7073:18)
[error]     at async main (file:///Users/tedks/Projects/mysten/sui/node_modules/prettier/internal/cli.mjs:7279:5)
[error]     at async Module.run (file:///Users/tedks/Projects/mysten/sui/node_modules/prettier/internal/cli.mjs:7225:5)
